content/200-orm/200-prisma-client/100-queries/090-raw-database-access/050-raw-queries.mdx

```diff
@@ -176,7 +186,7 @@ The `$queryRawUnsafe` method allows you to pass a raw string (or template string
 
 If you use this method with user inputs (in other words, `SELECT * FROM table WHERE columnx = ${userInput}`), then you open up the possibility for SQL injection attacks. SQL injection attacks can expose your data to modification or deletion.<br /><br />
 
-We strongly advise that you use the `$queryRaw` query instead. For more information on SQL injection attacks, see the [OWASP SQL Injection guide](https://www.owasp.org/index.php/SQL_Injection).
+Wherever possible you should use the `$queryRaw` query instead, taking into account that this method can also be made vulnerable in certain circumstances. For more information, see the [SQL Injection](#sql-injection) section below.
```

Comment:
This wording does not inspire confidence in the functionality. Can we instead say that it has to be used teh correct and safe way, and that one has to ensure not use it in unsafe ways (and then the link provides these example).